python/mxnet/_numpy_op_doc.py

```diff
@@ -19,7 +19,6 @@
 
 """Doc placeholder for numpy ops with prefix _np."""
 
-
```

Comment:
do not remove such lines, we require 2 lines between Python functions.

Reply:
Fixed.